better_exceptions/color.py

```diff
@@ -6,20 +6,45 @@
 
 from __future__ import absolute_import
 
+import codecs
 import errno
 import os
 import struct
 import sys
 
 from .context import PY3
-from .encoding import to_byte as _byte
 
 
 STREAM = sys.stderr
+ENCODING = getattr(STREAM, "encoding", None) or "ascii"
```

Comment:
Better to default to utf-8 IMO.

Reply:
@Qix- Yeah, I think you are right, that would avoid possible `UnicodeEncodeError`.